content/docs/observable.md

```diff
@@ -0,0 +1,202 @@
+# Observable
+
+> [tip box yellow] This functionality was added in v2.0.0-alpha.202 and is still experimental.
+
+### TLDR
+Imba provides MobX-like reactivity implemented as decorators.
+- Mark class and tag members as observable with the `@observable` decorator.
```

Comment:
I'm not sure what this means, "Mark class and tag members" Is this two things like this?

1. mark classes
2. tag members

If so what does it mean to "mark a class"? It seems that you only need to add these decorators to members, not to do anything specific to the class, right?